W-23025071: Correct Anypoint Monitoring protocol to HTTPS (not Lumber… (#1712)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/install-self-managed-network-configuration.adoc

|===
| Port | Protocol | Hostnames | Description
| 443 | AMQP over WebSockets | `transport-layer.prod.cloudhub.io` | Runtime Fabric message broker for interaction with the control plane.
| 443 (v2.6.22 and later) | HTTPS | `us1.ingest.mulesoft.com` | xref:use-anypoint-monitoring.adoc#enable-anypoint-monitoring[Anypoint Monitoring] agent for Runtime Fabric.
| 443 (pre-2.6.22, legacy) | TCP (Lumberjack) | `dias-ingestor-router.us-east-1.prod.cloudhub.io` | xref:use-anypoint-monitoring.adoc#enable-anypoint-monitoring[Anypoint Monitoring] agent for Runtime Fabric.

> *Required only for RTF versions prior to 2.6.22. After upgrading to RTF 2.6.22 or later and redeploying all applications, this endpoint is no longer used.*

Reviewer: ```suggestion
*Required only for Runtime Fabric versions earlier than 2.6.22. After upgrading to Runtime Fabric 2.6.22 or later and redeploying all applications, this endpoint is no longer used.*
```

| 5044 (legacy) | TCP (Lumberjack) | `dias-ingestor-nginx.prod.cloudhub.io` | xref:use-anypoint-monitoring.adoc#enable-anypoint-monitoring[Anypoint Monitoring] agent for Runtime Fabric.

> *As of Runtime Fabric version 1.8.50, this port is not strictly required. It will be deprecated in the future.*

Reviewer: ```suggestion
*Starting with Runtime Fabric version 1.8.50, this port is not required. It will be deprecated in the future.*
```

|===
| Port | Protocol | Hostnames | Description
| 443 | AMQP over WebSockets | `transport-layer.prod-eu.msap.io` | Runtime Fabric message broker for interaction with the control plane.
| 443 (v2.6.22 and later) | HTTPS | `eu1.ingest.mulesoft.com` | xref:use-anypoint-monitoring.adoc#enable-anypoint-monitoring[Anypoint Monitoring] agent for Runtime Fabric.
| 443 (pre-2.6.22, legacy) | TCP (Lumberjack) | `dias-ingestor-router.eu-central-1.prod-eu.msap.io` | xref:use-anypoint-monitoring.adoc#enable-anypoint-monitoring[Anypoint Monitoring] agent for Runtime Fabric.

> *Required only for RTF versions prior to 2.6.22. After upgrading to RTF 2.6.22 or later and redeploying all applications, this endpoint is no longer used.*

Reviewer: ```suggestion
*Required only for Runtime Fabric versions earlier than 2.6.22. After upgrading to Runtime Fabric 2.6.22 or later and redeploying all applications, this endpoint is no longer used.*
```

| 5044 (legacy) | TCP (Lumberjack) | `dias-ingestor-nginx.prod-eu.msap.io` | xref:use-anypoint-monitoring.adoc#enable-anypoint-monitoring[Anypoint Monitoring] agent for Runtime Fabric.

> *As of Runtime Fabric version 1.8.50, this port is not strictly required. It will be deprecated in the future.*

Reviewer: ```suggestion
*Starting with Runtime Fabric version 1.8.50, this port is not required. It will be deprecated in the future.*
```
